src/server/services/pickup/configure-server.js

```diff
@@ -0,0 +1,107 @@
+import Rcon from 'modern-rcon';
+import debug from 'debug';
+import fs from 'fs';
+import sleep from 'await-sleep';
+
+const log = debug('TF2Pickup:gameserver');
+
+/**
+ * Change server map.
+ *
+ * @param {Object} connection - RCON connection object to server.
+ * @param {Object} map - Map name.
+ */
+async function changeMap(connection, map) {
+  const maplist = await connection.send('maps *');
+
+  if (maplist.indexOf(map) === -1) {
+    throw new Error(`Server does not have map ${map}`);
+  } else {
+    log(`Changing server map to ${map}`);
+    await sleep(3 * 1000);
+    await connection.send(`changelevel ${map}`);
+  }
+}
+
+/**
+ * Change server config.
+ *
+ * @param {Object} connection - RCON connection object to server.
+ * @param {String} cfg - CFG file to execute.
+ */
+function executeConfig(connection, cfg) {
+  const rootPath = process.cwd();
+  const configPath = `${rootPath}/node_modules/@tf2-pickup/tf2-configs/dist/${cfg}.cfg`;
+  const configFile = fs.readFileSync(configPath, 'utf8');
+  const configLines = configFile.split('\n');
+
+  configLines.forEach(async (line) => {
```

Comment:
We will need to use a for loop here because forEach doesn't work with async functions